Apps/Common/SIMSolver.h

```diff
@@ -48,8 +48,8 @@ template<class T1> class SIMSolverStat : public SIMadmin
   //! \param[in] saveInterval The stride in the output file
   void handleDataOutput(const std::string& hdf5file, int saveInterval = 1)
   {
-    if (IFEM::getOptions().discretization < ASM::Spline && !hdf5file.empty())
-      IFEM::cout <<"\n  ** HDF5 output is available for spline discretization"
+    if (IFEM::getOptions().discretization < ASM::Lagrangian && !hdf5file.empty())
+      IFEM::cout <<"\n  ** HDF5 output is available for spline/lagrangian discretization"
```

Comment:
This will only eliminate the Triangular discretization, which I guess was not the intention. With the support of Lagrange, you can skip the entire check, I think.

Reply:
the intention was to remove spectral, but yeah, i messed up reading the enum.